src/sentry/utils/services.py

```diff
@@ -163,9 +163,12 @@ class ServiceDelegator(Service):
       request is rejected and the future will raise ``Queue.Full`` when
       attempting to retrieve the result.)
     - The ``callback_func`` is called after all futures have completed, either
-      successfully or unsuccessfully. The function is parameters are the method
-      name, calling arguments (as returned by ``inspect.getcallargs``) and a
-      mapping of backend name to ``Future`` objects.
+      successfully or unsuccessfully. The function parameters are:
+      - the method name (as a string),
+      - the calling arguments (as returned by ``inspect.getcallargs``),
+      - the backend names (as returned by the selector function),
+      - a list of results (as either a ``Future``, or ``None`` if the backend
```

Comment:
I changed this from a map, since I realized that a backend could hypothetically show up twice in the result of the selector (what I really want here is a ordered set, but using a list is simpler so I'm just accommodating that here.)